src/main/java/com/microsoft/azure/management/cosmosdb/samples/CreateCosmosDBWithEventualConsistency.java

```diff
@@ -106,34 +106,28 @@ public static boolean runSample(Azure azure, String clientId) {
         return false;
     }
 
-    private static void createDBAndAddCollection(String masterKey, String endPoint) throws DocumentClientException {
+    private static void createDBAndAddCollection(String masterKey, String endPoint) throws CosmosException {
         try {
-            DocumentClient documentClient = new DocumentClient(endPoint,
-                    masterKey, ConnectionPolicy.GetDefault(),
-                    ConsistencyLevel.Session);
+            CosmosClient cosmosClient = new CosmosClientBuilder()
+                .endpoint(endPoint)
+                .key(masterKey)
+                .consistencyLevel(ConsistencyLevel.SESSION).buildClient();
 
-            // Define a new database using the id above.
-            Database myDatabase = new Database();
-            myDatabase.setId(DATABASE_ID);
 
-            myDatabase = documentClient.createDatabase(myDatabase, null)
-                    .getResource();
+            cosmosClient.createDatabase(DATABASE_ID);
 
-            System.out.println("Created a new database:");
-            System.out.println(myDatabase.toString());
+            CosmosDatabase cosmosDatabase = cosmosClient.getDatabase(DATABASE_ID);
 
-            // Define a new collection using the id above.
-            DocumentCollection myCollection = new DocumentCollection();
-            myCollection.setId(COLLECTION_ID);
+            System.out.println("Created a new database:");
+            System.out.println(cosmosDatabase.toString());
 
             // Set the provisioned throughput for this collection to be 1000 RUs.
-            RequestOptions requestOptions = new RequestOptions();
-            requestOptions.setOfferThroughput(1000);
+            Integer throughput = 1000;
+            ThroughputProperties properties = ThroughputProperties.createAutoscaledThroughput(throughput);
```

Comment:
Just a heads up, you are using autoscaling feature of Cosmos here. Is this intentional?